### ISSUE TITLE
Wrong SHA256 checksum in PathCopyCopy17.0.exe.sha256 file?

### ISSUE THREAD

**Issue by goozak** [clechasseur/pathcopycopy#82]
According to [http://onlinemd5.com/](http://onlinemd5.com/) , the checksum in file `PathCopyCopy17.0.exe.sha256` is different from the one calculated for file `PathCopyCopy17.0.exe`.

That website (_with SHA-256 option selected_) usually works for checking checksums, so I'm guessing it's the value in file `PathCopyCopy17.0.exe.sha256` that is incorrect...?

**Comment by clechasseur** [comment]
I tried recalculating the SHA-256 of the executable using two different programs: Hashcheck and 7-zip. I also tried the checksum generator at https://defuse.ca/checksums.htm. All three gave me this value:

5ac9a3bcb1c5f7de9225a6a984e9eb90b2c1c101fcb6c6e20f725002b54ed22f

...which is the value found in the `PathCopyCopy17.0.exe.sha256` file published.

Your website, however, gives me

7E74AD3FB2078716725D5EAA2976FC915557AF5CA2AD8EFD22C9A13C835290AC

This is weird. Anyone knows what is the most "authoritative" SHA-256 generator available? I'd be inclined to think that it's your website that's wrong since all the other generators agree with each other...

**Comment by goozak** [comment]
I've used OnlineMD5 to check PHP and Apache setups downloads for a long time and it worked fine up until this executable.  I guess something in the file data is tripping it up.  Since there are no contact link on there, I'll just dump it.

Found this other one (good for bigger files) [https://md5file.com/calculator](https://md5file.com/calculator), which gives me the correct hash (yours).

Sorry for the false alarm!
